app/config/config_dev.yml

```diff
@@ -13,6 +13,12 @@ monolog:
             type:  stream
             path:  "%site.path_www%/var/logs/%kernel.environment%.log"
             level: debug
+        dockerstdout:
+            type: stream
+            path: php://stdout
+            level: debug
+            formatter: monolog.full_trace_formatter
+            channels: [!event]
```

Comment:
this also works without docker?

Reply:
Yeah I guess, only in dev mode. Do you mean running commands other than in docker could be affected?